### PR TITLE
Fixed iOS fonts not loading, added missing dependency, added check for value before accessing

### PR DESCRIPTION
**Pull request by feelingweird**
fixed iOS issue where fontFamily had more than one font family.
([UIFont fontNamesForFamilyName:familyName].count == 0) (ReactNative RCTConvert.m) blows up..
Should only need one font anyway as fonts are shipped.

fixed issue with value.indexOf blowing up on undefined value.

updated dev dependencies for missing babel plugin.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "del": "^2.0.2",
     "gulp": "^3.9.0",
     "gulp-babel": "^6.1.0",
-    "vinyl-paths": "^2.0.0"
+    "vinyl-paths": "^2.0.0",
+    "babel-preset-es2015": "^6.3.13"
```
Comment (codenamev) [inline package.json:22]: why is this dependency needed?

Comment (feelingweird) [inline package.json:22]: I believe it didn't compile without it, was a couple of weeks ago now,

**Comment by mlabrum** [comment]
any update :)

**Comment by Ehesp** [comment]
Sorry crazy busy at the moment.

If you wish to submit a PR onto the main repo to link to this git package instead of the NPM repo I'll merge.

**Comment by feelingweird** [comment]
Hey, 
I know how that is, throw a couple of kids in the mix to make it even more hectic :) 
i saw it was merged, did you still want me to submit that PR onto the main repo?

thanks,
kle

**Pull request merged**